src/sentry/scripts/similarity/index.lua

```diff
@@ -92,87 +119,188 @@ function table.izip(...)
     return result
 end
 
-function table.slice(t, start, stop)
```

Comment:
Might bring this back in place of `table.trim` because the name is more accurate. There's not a great way to actually trim a table in place in Lua that I know of/can think of.

Reply:
751a21b7e32d53d6648ea1740720eec62f2fb99c